props.json

```diff
@@ -1,4 +1,15 @@
 {
+  "1.4": {
+    "whatsNewUrl": "https://wolvic.com/blog/release_1.4",
+    "environments": [
+      {
+        "value": "skybox_r90",
+        "title": "Skybox R90",
```

Comment:
Please use another name as suggested in some other PR. We can call it "Flinstones" or Wolvic 1.4 if you don't want to be supercreative :)

Reply:
Done